public/app/features/logs/components/panel/LogList.tsx

```diff
@@ -38,8 +36,6 @@ import {
   storeLogLineSize,
 } from './virtualization';
 
-export type GetFieldLinksFn = (field: Field, rowIndex: number, dataFrame: DataFrame) => Array<LinkModel<Field>>;
```

Comment:
It's great that you found and used this type, which has been introduced for the new logs panel that is still in development 👍